rosbag2_cpp/src/rosbag2_cpp/info.cpp

```diff
@@ -17,8 +17,11 @@
 #include <filesystem>
 #include <unordered_map>
 #include <unordered_set>
+#include <vector>
+#include <memory>
 #include <stdexcept>
 #include <string>
+#include <utility>
```

Comment:
Very minor, but these could be ordered alphabetically. Same for all other files.

Reply:
Okay, fixed in the https://github.com/ros2/rosbag2/pull/1924/commits/b6984d8f711d151113d2914c7a8aa3c5a1f4b548

Reply:
Other files also have unordered includes.

Reply:
And I didn't necessarily ask to order the existing includes, just the ones you're adding here